pylib/Stages/Reporter/TextFile.py

```diff
@@ -87,13 +93,10 @@ def execute(self, log, keyvals, testDef):
                 except KeyError:
                     pass
                 if 0 != lg['status']:
```

Comment:
if you change this to `if 'status' in lg and lg['status'] != 0:`, and then you can remove the enclosing try/except to save on some indentation and improve readability.
Generally, it is probably a good idea to minimize the try/excepts all over the place for the sake of maintainability and readability.

Reply:
If `status` is guaranteed to be in `lg`, you don't even need to do that, you can just leave the if as-is and remove the try/except.

Reply:
I understand you don't like the overuse of try/accepts, however I am not adding these to the code they already existed beforehand.

I will try to reformat the code to remove try/accepts along with the other changes I made to ExecuteCmd and TextFile

Reply:
Just to be fair, I'm almost certainly the one who wrote that code. IIRC, the try/accept code was in there because at that time we hadn't required the values to be present, and hadn't settled on them being a list. Things were pretty unstable back then 😄 

I have no objection to removing them now that things have settled down.